### PR TITLE
Launch trending searches for Canada; switch all countries to use the clustering UI.

### PR DESCRIPTION
**Pull request by mimisun**

### CODE REVIEW

**Review by Jelena0921 — APPROVED** [review]
LGTM :)

**Review by RalphMorita — APPROVED** [review]
We need the localization on the tooltips and titles for the top queries/clusters but otherwise LGTM

File: public/lang/fr.json [hunk 1]
```diff
@@ -99,7 +99,7 @@
         "interest": "Popularité",
         "hi": "Haut",
         "lo": "Bas",
-        "change": "Modification"
+        "change": "Variation"
```
Comment (mimisun) [inline public/lang/fr.json:102]: I changed the translation to a shorter one so that it would fit in the UI.

**Review by RalphMorita — APPROVED** [review]
LGTM

**Pull request merged**